subsys/bluetooth/mesh/solicitation.c

```diff
@@ -190,13 +190,14 @@ void bt_mesh_sol_recv(struct net_buf_simple *buf, uint8_t uuid_list_len)
 	if (bt_mesh_gatt_proxy_get() == BT_MESH_GATT_PROXY_ENABLED ||
 	    bt_mesh_priv_gatt_proxy_get() == BT_MESH_GATT_PROXY_ENABLED ||
 	    bt_mesh_od_priv_proxy_get() == 0) {
+		LOG_DBG("Not soliciting");
 		return;
 	}
 
 	/* Get rid of ad_type that was checked in bt_mesh_scan_cb */
 	type = net_buf_simple_pull_u8(buf);
 	if (type != BT_DATA_UUID16_SOME && type != BT_DATA_UUID16_ALL) {
-		LOG_ERR("Invalid type 0x%x, expected 0x%x or 0x%x",
+		LOG_DBG("Invalid type 0x%x, expected 0x%x or 0x%x",
```

Comment:
Elsewhere in the host I think we've tried to use WARN log level for potentially invalid external data, whereas ERR is reserved for local failures. Is there a good reason to use DBG instead of WARN here (and elsewhere in the file)?

Reply:
Correction, WRN level (we used to have `BT_WARN()` but now it's `LOG_WRN()`.

Reply:
Actually, it shouldn't happen because it is pre-parsed here: https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/mesh/adv.c#L338-L346
IMHO this checking is redundant at all. However, this is bugfix not feature refactoring.

Reply:
Sure, but I don't think the log level changes are needed for the fix, right? They could be in a different commit or PR.

Reply:
They are very useful. Right now, without added debug messages, mesh device just silently ignores everything what PTS toolchain sends to it even I enable debugging of proxy, models etc.
If you insist, I'll add them as a separate commit in this PR. The maintenance and bugfixing of proxy solicitation functionality are much complex without them.